src/site/stages/build/drupal/graphql-tome-differences/taxonomy_term.md

```diff
@@ -0,0 +1,120 @@
+# taxonomy_term
+
+Tome returned **150** records.
+
+**This does not seem like it's used currently.**
```

Comment:
I think this might actually match the `fieldAdministration` object in `pages.json`

Reply:
Oh nice find! Could you post the `fieldAdministration` object here from your `pages.json`? I'll add it 👍